src/hypothesis/extra/datetime.py

```diff
@@ -123,3 +123,34 @@ def times(allow_naive=None, timezones=None):
 
 def datetime_to_time(dt):
     return dt.timetz()
+
+
+class TimedeltaStrategy(SearchStrategy):
+
+    def __init__(self, min_value=dt.timedelta.min, max_value=dt.timedelta.max):
+        assert type(min_value) == dt.timedelta, 'min_value must be a timedelta'
+        assert type(max_value) == dt.timedelta, 'min_value must be a timedelta'
```

Comment:
You can probably do away with the assert message in both cases – it doesn’t tell you anything beyond seeing the assert line has failed. And as demonstrated here, it can’t get out of sync with the code. :-)